plate.py

```diff
@@ -0,0 +1,63 @@
+class Dimensions():
+
+    def __init__(self, number_of_rows, number_of_columns):
+        """Initiate attributes for a plate"""
+        self.number_of_rows = number_of_rows
+        self.number_of_columns = number_of_columns
+
+class PlateCell():
+
+    def __init__(self, cell_number, dimensions):
+        self.cell_number = int(cell_number)
+        self.dimensions = dimensions
+
+    def calculate_row_and_column(self):
+        """Calculate row and column based on cell number"""
+        if self.cell_number > self.dimensions.number_of_rows * self.dimensions.number_of_columns:
+            raise AttributeError("Cell number is too big for current dimension!")
```

Comment:
Yep, but you could've detected this problem in the constructor. The sooner we detect errors - the better.

Not sure if this should be an `AttributeError` though. Sounds more like `ValueError`